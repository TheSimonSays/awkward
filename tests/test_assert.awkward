assert(1 == 1)
assert(2 + 2 == 4)
print("before assert")
assert(3 < 2, "(3 < 2) is wrong")
print("after assert")
