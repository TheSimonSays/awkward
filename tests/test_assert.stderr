AssertionError: (3 < 2) is wrong
